.eslintrc.js

```diff
@@ -249,6 +249,14 @@ module.exports = {
                         message: "Please don't declare enums, use union types instead.",
                     },
                 ],
+                'no-restricted-properties': [
+                    'error',
+                    {
+                        object: 'Image',
+                        property: 'getSize',
+                        message: 'Usage of Image.getImage is restricted. Please use the `react-native-image-size`.',
+                    },
+                ],
```

Comment:
This error can be ignored by changing the import name like this:
`import {Image as Image2} from '...'`

I couldn't find a way to block this one as well.